test/onnx_import.in.cpp

```diff
@@ -2227,6 +2227,101 @@ NGRAPH_TEST(onnx_${BACKEND_NAME}, import_non_existing_file)
+    // W
+    std::vector<float> in_w{0.1f, 0.2f, 0.3f, 0.4f, 1.f, 2.f, 3.f, 4.f, 10.f, 11.f, 12.f, 13.f};
+    // R
+    std::vector<float> in_r(4 * hidden_size * hidden_size, 0.1f);
+    // B
+    std::vector<float> in_b(8 * hidden_size, 0.0f);
+
+    std::vector<int> in_seq_lengths{1, 2};
+
+    std::vector<float> out_y_data{0.28828835f,
+                                  0.36581863f,
+                                  0.45679406f,
+                                  0.34526032f,
+                                  0.47220859f,
+                                  0.55850911f,
+                                  0.f,
+                                  0.f,
+                                  0.f,
+                                  0.85882828f,
+                                  0.90703777f,
+                                  0.92382453f};
+
+    std::vector<float> out_y_h_data{
+        0.28828835f, 0.36581863f, 0.45679406f, 0.85882828f, 0.90703777f, 0.92382453f};
+
+    std::vector<float> out_y_c_data{
+        0.52497941f, 0.54983425f, 0.5744428f, 1.3249796f, 1.51063104f, 1.61451544f};
+
+    Outputs expected_output;
+
+    expected_output.emplace_back(out_y_data);
+    expected_output.emplace_back(out_y_h_data);
+    expected_output.emplace_back(out_y_c_data);
+
+    auto backend = ngraph::runtime::Backend::create("${BACKEND_NAME}");
+    auto parameters = function->get_parameters();
+
+    EXPECT_TRUE(parameters.size() == parameters_cout);
+
+    std::vector<std::shared_ptr<ngraph::runtime::Tensor>> arg_tensors;
+
+    auto add_tensor = [&arg_tensors, &backend](const std::vector<float>& v,
+                                               const std::shared_ptr<ngraph::op::Parameter>& p) {
+        auto t = backend->create_tensor(p->get_element_type(), p->get_shape());
+        copy_data(t, v);
+        arg_tensors.push_back(t);
+    };
+
+    add_tensor(in_x, parameters.at(0));
+    add_tensor(in_w, parameters.at(1));
+    add_tensor(in_r, parameters.at(2));
+    add_tensor(in_b, parameters.at(3));
+
+    auto t_in_seq_lengths =
+        backend->create_tensor(parameters.at(4)->get_element_type(), parameters.at(4)->get_shape());
+    copy_data(t_in_seq_lengths, in_seq_lengths);
+    arg_tensors.push_back(t_in_seq_lengths);
+
+    auto results = function->get_results();
+    std::vector<std::shared_ptr<ngraph::runtime::Tensor>> result_tensors(results.size());
+
+    for (size_t i = 0; i < results.size(); ++i)
+    {
+        result_tensors.at(i) =
+            backend->create_tensor(results.at(i)->get_element_type(), results.at(i)->get_shape());
+    }
+
+    auto handle = backend->compile(function);
+    handle->call_with_validate(result_tensors, arg_tensors);
+
+    Outputs outputs;
+    for (auto rt : result_tensors)
+    {
+        outputs.push_back(read_vector<float>(rt));
+    }
+
+    EXPECT_TRUE(outputs.size() == expected_output.size());
+    for (std::size_t i{0}; i < expected_output.size(); ++i)
```

Comment:
Just a very low importance comment: `std::size_t -> size_t` for consistency

Reply:
Actually I'm adding this prefix few lines above. Within this file we're using `std::` prefixes.